src/components/App/App.jsx

```diff
@@ -63,58 +76,82 @@ function App() {
 
   return (
     <AppGrid>
-      <Countdown minutes={minutes} seconds={seconds} />
-      <LengthButtonGrid>
-        <TimerLength
+      <CountdownGrid>
+        <Countdown minutes={minutes} seconds={seconds} />
+        {isActive ? (
+          <EndTime endHours={endHours} endMinutes={endMinutes} />
+        ) : (
+          ''
+        )}
+      </CountdownGrid>
+      <ConfigurationGrid>
+        <CountdownLength
           onClick={handleLengthShort}
           disabled={isActive}
           selected={!lengthLong}
         >
           25:00
-        </TimerLength>
-        <TimerLength
+        </CountdownLength>
+        <CountdownLength
           onClick={handleLengthLong}
           disabled={isActive}
           selected={lengthLong}
         >
           50:00
-        </TimerLength>
-      </LengthButtonGrid>
-      {!isActive ? (
-        <PlayButton role="button" onClick={handleStart} />
-      ) : (
-        <StopButton role="button" onClick={handleStop} />
-      )}
+        </CountdownLength>
+      </ConfigurationGrid>
+      <StartStopGrid>
+        {!isActive ? (
+          <PlayButton role="button" onClick={handleStart} />
+        ) : (
+          <StopButton role="button" onClick={handleStop} />
+        )}
+      </StartStopGrid>
     </AppGrid>
   )
 }
 
 export default App
 
-const AppGrid = styled.div`
+const AppGrid = styled.main`
   display: grid;
   position: fixed;
   left: 0;
   top: 0;
   width: 100%;
   height: 100%;
-  place-content: center;
-  place-items: center;
-  gap: 40px;
+  grid-template-rows: 1fr 1fr 100px;
 `
 
-const LengthButtonGrid = styled.div`
+const CountdownGrid = styled.section`
   display: grid;
-  grid-template-columns: 1fr 1fr;
   gap: 10px;
+  grid-template-rows: 1fr 30px;
+  align-items: end;
+  justify-items: center;
 `
 
-const TimerLength = styled.button`
+const ConfigurationGrid = styled.section`
+  display: grid;
+  grid-template-columns: auto auto;
+  align-content: end;
+  justify-content: center;
+  gap: 10px;
+  padding: 20px;
+`
+
+const StartStopGrid = styled.section`
+  display: grid;
+  align-content: start;
+  justify-content: center;
+  padding: 10px;
+`
+
+const CountdownLength = styled.button`
   color: ${props => (props.selected ? '#52DFD1' : '#585858')};
   border: none;
-  width: fit-content;
-  height: fit-content;
   background-color: transparent;
```

Comment:
Is there a need for setting a transparent background? Otherwise you could remove this line.